static/app/actionCreators/dashboards.tsx

```diff
@@ -78,7 +80,7 @@ export function updateDashboard(
 ): Promise<DashboardDetails> {
   const data = {
     title: dashboard.title,
-    widgets: dashboard.widgets,
+    widgets: dashboard.widgets.map(widget => omit(widget, ['tempId'])),
```

Comment:
Hmmm, `omit()` is more readable 😀

```suggestion
    widgets: dashboard.widgets.map(({tempId: _, ...widget}) => widget),
```

Reply:
I'm going to leave it as-is then! I forgot about destructuring for this use case, but I also think `omit()` is more readable here